cmd/akamai-purger/main.go

```diff
@@ -24,40 +23,150 @@ import (
 	blog "github.com/letsencrypt/boulder/log"
 )
 
-// defaultQueueSize is the default akamai-purger queue size.
-const defaultQueueSize = 1000000
+const (
+	// TODO(#6003) remove entirely.
+	DeprecatedURLBatchSize = 33
+
+	// defaultQueueSize is the size of the queue when the configuration key
+	// 'maxQueueSize' is unspecified. A queue size of 1.25M cached OCSP
+	// responses, assuming 3 URLs per request, is about 6 hours of work using
+	// default settings.
+	defaultQueueSize         = 1250000
+	defaultResponsesPerBatch = 2
+	defaultPurgeBatchEvery   = time.Millisecond * 32
+
+	// akamaiBytesPerResponse is the total bytes of all 3 URLs of cached OCSP responses associated with
+	// each certificate.
```

Comment:
Is this precise, or an estimate? If the latter, indicate that. Also probably indicate what this is used for, because it isn't as obviously an configuration parameter like the three above are.

Reply:
I've fleshed this out quite a bit more. Please let me know if I've overdone it.